packages/api/lib/writeRecords/write-granules.js

```diff
@@ -895,6 +896,7 @@ const writeGranulesFromMessage = async ({
         error,
         pdrName,
         workflowStatus,
+        timestamp,
```

Comment:
Do we need to make this same change in `writeGranuleFromApi`? otherwise records created via the API won't have a timestamp, I think? 

Or we could update `generateGranuleApiRecord` to set a default and just allow it to be overwritten here.

Either way, wee probably should prove that records from `writeGranuleFromApi` have a `timestamp` since we're relying on it in th dashboard

Reply:
Yes this is probably the thing you explicitly called out yesterday when we were going over it, and I missed it and forgot to ask about it again.

Reply:
de345ec and 5392472

Reply:
Should we add a unit test proving that the createdAt, updatedAt, and timestamp get set properly for `writeGranuleFromApi`?